packages/frontend/src/features/metricsCatalog/components/MetricsCatalogColumnName.tsx

```diff
@@ -241,15 +259,19 @@ export const MetricsCatalogColumnName = forwardRef<HTMLDivElement, Props>(
                         variant="xs"
                         openDelay={300}
                     >
-                        <Highlight
-                            highlight={table.getState().globalFilter || ''}
-                            c="dark.9"
-                            fw={500}
-                            fz="sm"
-                            lh="150%"
+                        <UnstyledButton
+                            onClick={() => handlePeekMetric(row.original)}
```

Comment:
is it okay to have it exposed like this or should we hide it better?

Reply:
I think it's ok from chatting with @PriPatel